assets/js/controllers/user_view_ctrl.js

```diff
@@ -0,0 +1,11 @@
+app.controller("userViewCtrl", ["$scope", "User", "AuthService", function($scope, User, AuthService){
+  AuthService.currentUser(function(user) {
+    if (user.administrating.length !== 0) {
```

Comment:
Should be: ```$scope.admin = (user.administrating.length !== 0);```

When the array has a length, evaluates to true which enables the admin stuff. When array has a length of 0, evalutes to false, student view is then used.